src/main/java/org/opensearch/sdk/NettyTransport.java

```diff
@@ -40,9 +40,18 @@
 
 public class NettyTransport {
     private static final String NODE_NAME_SETTING = "node.name";
+    private final ExtensionsRunner extensionsRunner;
     private final TransportInterceptor NOOP_TRANSPORT_INTERCEPTOR = new TransportInterceptor() {
     };
 
+    /**
+     *
```

Comment:
Nit: whitespace.  

Non-nit: this javadoc isn't very explanatory.  Something like "Instantiate this object with a reference to the ExtensionsRunner."   Also the `@param` tag description doesn't describe the ExtensionsRunner.